app/lib/location_sessions_factory.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
 class LocationSessionsFactory
-  def initialize(location, academic_year:)
+  def initialize(
+    location,
+    academic_year:,
+    sync_patient_teams_immediately: false
```

Comment:
```suggestion
    sync_patient_teams_now: false
```

(Or `sync_patient_teams_later: true`)

Just to be consistent about the naming, and gives a hint that under the hood it's calling `perform_now` or `perform_later`.